open_rarity/models/token_features.py

```diff
@@ -0,0 +1,15 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class TokenFeatures:
```

Comment:
Token Features sounds like features for a token but it's for ranking - 
Could we do something like `RankingFeatures` or `RankingInputs` or `TokenRankingInputs` if we wanted to be more clear but verbose haha

Reply:
`TokenRankingFeatures`

Reply:
will it work?